staging/src/k8s.io/csi-translation-lib/plugins/rbd.go

```diff
@@ -294,7 +294,10 @@ func composeMigVolID(mons string, pool string, image string) string {
 	clusterIDInHandle := md5.Sum([]byte(mons))
 	clusterField := monsPfx + fmt.Sprintf("%x", clusterIDInHandle)
 	poolHashInHandle := hex.EncodeToString([]byte(pool))
-	imageHashInHandle := strings.Split(image, defaultIntreeImagePfx)[1]
+	imageHashInHandle := image
+	if strings.Contains(image, defaultIntreeImagePfx) {
```

Comment:
`strings.StartsWith()`?

Reply:
Looks like we are good with only `TrimPrefix` as it supposed to give the image name as it is if there is no prefix.  I have used `TrimPrefix` alone @jsafrane